packages/docs/getting-started/running-a-validator-in-rc1.md

```diff
@@ -153,40 +153,131 @@ The setup of RC1 is similar to the new Baklava network and the deployment timeli
 
 ```bash
 export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:rc1
 export NETWORK_ID=42220
-export CELO_VALIDATOR_SIGNER_ADDRESS=<YOUR-VALIDATOR-SIGNER-ADDRESS>
 ```
 
-Please use the Validator signer address that you submitted through your Gist file. It is included in the [genesis Validator set](https://github.com/celo-org/celo-monorepo/blob/asaj/rc1/packages/celotool/genesis_rc1.json). If you plan to run additional validators once the network is up and running you will need to create additional validator signer addresses.
-
 ### Pull the Celo Docker image
 
-In all the commands we are going to see the `CELO_IMAGE` variable to refer to the Docker image to use. Now we can get the Docker image on your Validator and Proxy machines:
+In all the commands we are going to see the `CELO_IMAGE` variable to refer to the Docker image to use. Now we can get the Docker image on all machines:
 
 ```bash
 docker pull $CELO_IMAGE
 ```
 
 The `us.gcr.io/celo-testnet/celo-node:rc1` image contains the [genesis block](https://github.com/celo-org/celo-monorepo/blob/asaj/rc1/packages/celotool/genesis_rc1.json) in addition to the Celo Blockchain binary.
 
+### Create the Validator and Validator Group accounts
+
+First, you'll need to generate account keys for your Validator and Validator Group.
+
+{% hint style="danger" %}
+These keys will become the beneficiary addresses of your `ReleaseGold` contracts, and thus should be handled with care.
+
+Store and back these keys up in a secure manner, as there will be no way to recover if them if lost or stolen.
+{% endhint %}
+
+```bash
+# On your local machine
+mkdir celo-accounts-node
+cd celo-accounts-node
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE account new
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE account new
+```
+
+This will create a new keystore in the current directory with two new accounts.
+Copy the addresses from the terminal and set the following environment variables:
+
+```bash
+# On your local machine
+export CELO_VALIDATOR_GROUP_ADDRESS=<YOUR-VALIDATOR-GROUP-ADDRESS>
+export CELO_VALIDATOR_ADDRESS=<YOUR-VALIDATOR-ADDRESS>
+```
+
+### Start your Accounts node
+
+Next, we'll run a node on your local machine so that we can use these accounts to lock Celo Gold and authorize the keys needed to run your validator. To do this, we need to run the following commands, which fetch the genesis block and a list of other nodes in the network to connect to.
+
+```bash
+# On your local machine
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE init /celo/genesis.json
+export BOOTNODE_ENODES=`docker run --rm --entrypoint cat $CELO_IMAGE /celo/bootnodes`
+```
+
+To run the node:
+
+```bash
+# On your local machine
+docker run --name celo-accounts -it --restart always -p 127.0.0.1:8545:8545 -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin,personal --bootnodes $BOOTNODE_ENODES --gcmode=archive
```

Comment:
does this need an `--etherbase`? Saw it on other commands, but im not 100% clear what it does? I believe it specifies an account to use as the default for rewards. In this case maybe we want to specify one to collect any full node rewards?

cc @asaj

Reply:
not sure either. Also, I added `--gcmode=archive` -- @asaj is this still necessary?

Reply:
etherbase shouldn't be necessary for this node. `--gcmode=archive` isn't either, though there are some CLI commands that won't work without it. The best thing to do would be to fix those CLI commands, as an archive node is rather disk intensive

Reply:
addressed!